Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -179,6 +178,48 @@ public class PBXProjGenerator {
         )
         derivedGroups.append(productGroup)
 
+        let subprojectFileReferences: [PBXFileReference] = project.projectReferences.map { projectReference in
+            let projectPath = Path(projectReference.path)
+
+            return addObject(
+                PBXFileReference(
+                    sourceTree: .group,
+                    name: projectReference.name,
+                    lastKnownFileType: Xcode.fileType(path: projectPath),
+                    path: projectPath.normalize().string
+                )
+            )
+        }
+        if subprojectFileReferences.count > 0 {
+            let subprojectsGroups = addObject(
+                PBXGroup(
+                    children: subprojectFileReferences,
+                    sourceTree: .group,
+                    name: "Projects"
```

Comment:
Is adding all projects into a `Projects` group sufficient for most people? I could imagine people wanting to add their projects to a specific group.

Reply:
For my use case, it doesn't really matter. But I agree, I could see a use for it. How would you think that would work best? A new parameter under projectReferences?